styles/mixins/_atoms.scss

```diff
@@ -0,0 +1,35 @@
+@use 'sass:color';
+@use 'sass:list';
+
+@use '../variables' as var;
+
+$-color-border: var.$color-border;
+$-color-shadow: var.$color-shadow;
+
+/**
+ * Add border styles for all element sides (default) or a specific single side
+ *
+ * @param {'left'|'right'|'bottom'|'top'} [$side]
+ */
+@mixin border($side: null) {
+  $valid-sides: left, right, bottom, top;
+  $border-property: border;
+  @if list.index($valid-sides, $side) {
```

Comment:
Learning about the sass syntax, I found this check. I don't know if it is useful:

```
  @if not list.index($valid-sides, $side) {
    @error "#{$side} is not a valid side. Expected one of #{$valid-sides}.";
  }
```

Reply:
I also saw that example! I could add this, sure, if you think it would be useful!

Reply:
Actually, in reviewing this `mixin`, I think the current behavior is a little more straightforward as-is, given that the default behavior is to style a border for all the sides. Please keep speaking up, though, if you feel strongly about this.

Reply:
No, it is not necessary, thanks.